fighthealthinsurance/ml/ml_models.py

```
@@ -1715,41 +1739,39 @@ async def __infer(
                         try:
                             error_body = await response.text()
                             truncated_body = error_body[:500]
-                            logger.debug(
+                            logger.warning(
                                 f"HTTP {e.status} error from {api_base} for model {model}: body_prefix={truncated_body!r}"
                             )
                         except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)